build.ps1

```
@@ -16,31 +16,35 @@ if ($env:APPVEYOR_PULL_REQUEST_NUMBER)
 	# download nuget.exe
 	$sourceNugetExe = "https://dist.nuget.org/win-x86-commandline/latest/nuget.exe"
 	Invoke-WebRequest $sourceNugetExe -OutFile $targetNugetExe
 }
 
 msbuild /t:Restore,Pack ./src/NLog/ /p:targetFrameworks='"net46;net45;net35;netstandard2.0;netstandard2.1"' /p:VersionPrefix=$versionPrefix /p:VersionSuffix=$versionSuffix /p:FileVersion=$versionFile /p:ProductVersion=$versionProduct /p:Configuration=Release /p:IncludeSymbols=true /p:SymbolPackageFormat=snupkg /p:ContinuousIntegrationBuild=true  /p:EmbedUntrackedSources=true /p:PackageOutputPath=..\..\artifacts /verbosity:minimal /maxcpucount
-if (-Not $LastExitCode -eq 0)
+if (-not $LastExitCode -eq 0)
 	{ exit $LastExitCode }
 
-function create-package($packageName, $targetFrameworks)
+function Create-Package($packageName, $targetFrameworks)
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Create-Package' uses an unapproved verb.

(PSUseApprovedVerbs)